### PR TITLE
feat: automating `og:image` for `getFrameHtmlResponse` and `FrameMetadata`.

### PR DESCRIPTION
**Pull request by Zizzamia**
**What changed? Why?**
This relates to:
- https://github.com/coinbase/onchainkit/issues/103
- https://github.com/coinbase/onchainkit/issues/44

**Notes to reviewers**

**How has it been tested?**

### CODE REVIEW

File: src/core/getFrameHtmlResponse.ts [hunk 1]
```diff
@@ -19,8 +19,8 @@ function getFrameHtmlResponse({
   refreshPeriod,
   refresh_period,
 }: FrameMetadataType): string {
-  // Set the image metadata if it exists.
-  const imageHtml = image ? `  <meta property="fc:frame:image" content="${image}" />\n` : '';
+  const imageHtml = `  <meta property="fc:frame:image" content="${image}" />\n`;
+  const ogImageHtml = `  <meta property="og:image" content="${image}" />\n`;
```
Comment (spennyp) [inline src/core/getFrameHtmlResponse.ts:23]: From my testing, `og:title` is also required to have warpcast render the frame. 
Can just set to empty string to satisfy this.

Comment (spennyp) [reply src/core/getFrameHtmlResponse.ts:23]: Or, looks like they have some updated docs, and it seems maybe not necessary: https://docs.farcaster.xyz/reference/frames/spec#required-properties

Comment (Zizzamia) [reply src/core/getFrameHtmlResponse.ts:23]: Not sure i follow your point

Comment (spennyp) [reply src/core/getFrameHtmlResponse.ts:23]: You added the `og:image` metadata here, but not `og:title`.

From my testing, without `og:image` AND `og:title`, warpcast won't render a frame from a GET request. 

This works:
```html
<!DOCTYPE html>
<html>
    <head>
	<meta property="og:image" content="https://paperclip.a.pinggy.link/images/nouns-auction-house.png" />
	<meta property="og:title" content="title" />
	<meta property="fc:frame" content="vNext" />
	<meta property="fc:frame:image" content="https://paperclip.a.pinggy.link/images/nouns-auction-house.png" />
	<meta property="fc:frame:post_url" content="https://paperclip.a.pinggy.link/auction/nouns?" />
	<meta property="fc:frame:button:1" content="View auction!" />
	<meta property="fc:frame:button:1:action" content="post" />
    </head>
</html>
    
```

This doesn't work (only difference is that it is missing the `og:title` tag)
```html
<!DOCTYPE html>
<html>
    <head>
	<meta property="og:image" content="https://paperclip.a.pinggy.link/images/nouns-auction-house.png" />
	<meta property="fc:frame" content="vNext" />
	<meta property="fc:frame:image" content="https://paperclip.a.pinggy.link/images/nouns-auction-house.png" />
	<meta property="fc:frame:post_url" content="https://paperclip.a.pinggy.link/auction/nouns?" />
	<meta property="fc:frame:button:1" content="View auction!" />
	<meta property="fc:frame:button:1:action" content="post" />
    </head>
</html>
```

Comment (Zizzamia) [reply src/core/getFrameHtmlResponse.ts:23]: But the spec doesn't care about the title?!?

Is it the spec wrong?

Comment (spennyp) [reply src/core/getFrameHtmlResponse.ts:23]: Yea which is why I was confused, but I just retested.

It seems the spec is not accurate here. `og:title` definitely makes a difference.

Comment (Zizzamia) [reply src/core/getFrameHtmlResponse.ts:23]: Let me loop back this to V

Comment (Zizzamia) [reply src/core/getFrameHtmlResponse.ts:23]: But yeah, if it doesn't work, than yeah let me add `og:title`, and thank you for triple checking this.

Comment (spennyp) [reply src/core/getFrameHtmlResponse.ts:23]: Awesome, thanks!

Oh, and `content=""` (empty string) doesn't work lol (very finicky), so just put "title" or something...

Comment (spennyp) [reply src/core/getFrameHtmlResponse.ts:23]: Sorry for the nits lol

This is the last thing stopping us from moving over to use onchainkit 🙏!

Comment (Zizzamia) [reply src/core/getFrameHtmlResponse.ts:23]: 🤔 I guess is fair to let people decide, and than yeah maybe the default is something like `Frame title`

**Comment by Zizzamia** [comment]
@spennyp random question.
when you say `This doesn't work (only difference is that it is missing the og:title tag)`. Where it doesn't work, because I use that method with https://github.com/Zizzamia/a-frame-in-100-lines/blob/main/app/api/frame/route.ts#L28 and it works. Even without `og:image` and `og:title` stuff.

**Comment by spennyp** [comment]
> @spennyp random question. when you say `This doesn't work (only difference is that it is missing the og:title tag)`. Where it doesn't work, because I use that method with https://github.com/Zizzamia/a-frame-in-100-lines/blob/main/app/api/frame/route.ts#L28 and it works. Even without `og:image` and `og:title` stuff.

It’s when using getFrameHtmlResponse in a GET request instead of using page.tsx for the first response like here:
https://github.com/papercliplabs/frames/blob/main/src/app/auction/%5Bdao%5D/route.tsx#L6

**Comment by Zizzamia** [comment]
@spennyp oh I see your point now! Ok give me a few minutes and I will finish and merge this PR.

**Pull request merged**